core/src/main/res/values/colors.xml

```diff
@@ -12,8 +12,11 @@
   <color name="blue800">#1565c0</color>
   <color name="pink300">#f06292</color>
   <color name="actionModeBackground">#4285F4</color>
+  <color name="gray350">#FFD6D6D6</color>
   <color name="gray500">#9E9E9E</color>
   <color name="gray700">#616161</color>
+  <color name="gray850">#303030</color>
+  <color name="card_view_background">#FFFFFF</color>
```

Comment:
I don't like that `card_view_background` is not being applied to CardViews. This seems a likely candidate for a theme on CardViews.

There is in fact a theme attribute called `cardBackgroundColor`

Reply:
Okay. I will use the `cardBackgroundColor` and update the commit.